.github/workflows/upload_package.yml

```diff
@@ -1,53 +1,45 @@
-
 name: Release Package
 
 on:
   push:
     # Sequence of patterns matched against refs/tags
     tags:
       - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Manual Release
+        default: test
+        required: false
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: '3.x'
+    
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade build twine
-        python -m build
-        twine check --strict dist/*
+        pip install setuptools wheel twine jupyter urllib3 pandas pyyaml
+        python setup.py sdist bdist_wheel
```

Comment:
@jGaboardi does this line with setup.py still work with pyproject.toml? I thought you had to use `pip install .` ?

Reply:
See [here](https://github.com/pysal/spaghetti/blob/93cd5346eed24db92c65bc07eef74d6d78487903/.github/workflows/release_and_publish.yml#L34) in `spaghetti`'s release workflow.

Reply:
ah ok thanks. I snagged the last one from spopt

Reply:
Have not had enough time recently to sync everything 😰